packages/ngx-flicking/projects/ngx-flicking/src/lib/ngx-flicking.component.ts

```diff
@@ -4,9 +4,10 @@
  */
 
 // tslint:disable-next-line: max-line-length
-import NativeFlicking, { Plugin, FlickingOptions, withFlickingMethods, DEFAULT_OPTIONS, FlickingEvent, NeedPanelEvent, SelectEvent, ChangeEvent } from '@egjs/flicking';
+import NativeFlicking, { Plugin, FlickingOptions, withFlickingMethods, DEFAULT_OPTIONS, FlickingEvent, NeedPanelEvent, SelectEvent, ChangeEvent, VisibleChangeEvent } from '@egjs/flicking';
+import { counter } from '../../../../../../src/utils';
```

Comment:
If we're not targeting IE8, I think it's better to use another implementation of counter(or `range`, like python).
As native Flicking's counter is quite rough.

Reply:
Like,
```js
const counter = num => Array(num).map((_, i) => i));
// or 
const counter = num => [...Array(num).keys()];
```

Reply:
Array.keys also cannot support IE.

Reply:
I mean, **If we're not targeting IE8** in Angular.

Reply:
I've just found that angular support from IE9 ~ 😢 
https://angular.io/guide/browser-support
![image](https://user-images.githubusercontent.com/3255471/64413256-1ffb9d00-d0cc-11e9-9316-9aad52db3607.png)

Reply:
I know it's trivial, but the upper one can be used instead.
Because native Flicking's counter is like this.
```ts
// return [0, 1, ...., max - 1]
export function counter(max: number): number[] {
  const counterArray: number[] = [];
  for (let i = 0; i < max; i += 1) {
    counterArray[i] = i;
  }
  return counterArray;
}
```
It's made in the beginning of 3.0, when I had no clue whether to support IE8 or not.